client/components/Layout/FormEditScholarship.js

```diff
@@ -20,22 +28,25 @@ const FormEditScholarship = () => {
 		alert(JSON.stringify(e))
 	}
 
-	const formProps = { register, errors }
+	const formProps = { register, errors, getValues, setValue, control, watch }
 	return (
 		<Stack>
 			<FormControl component="form" onSubmit={handleSubmit(onSubmit)} sx={{ width: '100%' }}>
 				<Stack spacing={3} direction="column">
 					<TextFieldComponent name="Scholarship Name" {...formProps} />
```

Comment:
name should be the same as field name in model, use label instead if you want to customize